singleheader/README.md

```diff
@@ -0,0 +1,22 @@
+While in the ada main directory, using Python 3, type:
```

Comment:
This file will be included in the documentation page, if we don't explicitly exclude it. If we want to add this, let's add a descriptive header using `#` prefix.

Reply:
Right, but that should not be front-page. End-users should not ever have to do this.